resilio.md

```diff
@@ -0,0 +1,40 @@
+
+Now, one of the Partners needs to share the `Niteo` folder with you. They do this by opening up Resilio and clicking the `Share` button for the `Niteo` folder and sending you the `Read & Write` link . When you click the link, Resilio should start up and add the shared folder to your device. If this does not happen, open Resilio and click the plus icon to manually enter the link.
+
+It is important the you turn `Selective Sync` ON. This tells Resilio to only download filenames to your machine, without the actual data, to save disk space. When you then try to open any of these empty files in your file manager (Finder, Explorer, etc.), Resilio will download the data and start keeping the file in sync. On Linux, you have to go to Resilio's local Web UI to select which files you want to keep synced to your local machine.
+
+
+## Trialists
+
+Trialists get access to files on a need-to-have basis. Their mentor opens up Resilio, clicks the plus icon and selects
+`Share file` to share a file with the trialist.
+
+
+## Additional Folders
+
+Besides the `Niteo` folder, there are two more folders:
+* `NiteoPartners`: A store of sensitive documents, only Partners have access. Also an archive of all old projects & documents. Has the same folder structure as Intra, plus Archive (with the same substructure).
+* `NiteoFounders`: A few very sensitive files, such as encryption keys, that max 2 people should have access to. Has the same folder structure as Intra.
+* `Horus`: This is a `1_Projets/Kai/Horus` subfolder inside the `Niteo` folder that is it's own Resilio Share. AI servers (Horus, Osiris, Anubis) run Resilio and sync contents of the `Horus` folder to `/resilio` directories on their disks. This is used to share AI models between AI servers and researchers.
+
+
+## Always-online Peers
+
+Resilio is a peer-to-peer protocol, based on BitTorrent. This means that if no Niteans are online, you have no-one to sync data with. With Dropbox, you always sync into the cloud, even if you are the only team member currently online.
+
+To work around this limitation, we run two peers that are always online. One is our Somnus archive server, the other is
+Morpheus (the son of Somnus) which is only a Resilio peer. These peers have Selective Sync turned OFF, so they keep a copy of all data we have in Resilio, so even if we lose one, all data is replicated on the other (besides on our laptops). For security purposes, Somnus and Morpheus are located in separate physical locations.
+
+
+## Limitations
+
+* Up to 143 characters long filenames: We use Synology NAS devices to act as always-online peers. The data on these devices is stored on encrypted volumes. These volumes have a limite of up to 143 characters filenames.
```

Comment:
Spelling of `limite`

However, should we mention precisely what we use in handbook and I would suggest this change:

`Up to 140 chars for filenames (folder paths are not included in this limit). This limit is due to our resilio seeding server using encryptfs, see: https://unix.stackexchange.com/a/32834/8443`

I'll leave it to you whether we use the word `seed` instead of `peer` for the servers. ;)

Reply:
I agree seed is technically more correct, but I'd still go with peer to have less technical terms in one document. Handbooks should be readable by everyone.